Screen-Sharing-Video-Chat /css/app.css

```diff
@@ -0,0 +1,76 @@
+body, html {
+    background-color: gray;
+    height: 100%;
+}
+
+#videos {
+    position: relative;
+    width: 100%;
+    height: 100%;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+#subscriber {
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 10;
+}
+
+#publisher {
+    position: absolute;
+    width: 360px;
+    height: 240px;
+    bottom: 20px;
+    left: 10px;
+    z-index: 100;
+    border: 3px solid white;
+    border-radius: 3px;
+}
+
+#publisher {
+    position: absolute;
+    width: 360px;
+    height: 240px;
+    bottom: 20px;
+    left: 10px;
+    z-index: 100;
+    border: 3px solid white;
+    border-radius: 3px;
+}
+
+#screen {
+    position: absolute;
+    width: 360px;
+    height: 240px;
+    bottom: 20px;
+    right: 10px;
+    z-index: 100;
+    border: 3px solid white;
+    border-radius: 3px;
+}
+
+.toggle-button-on {
+    background-color: green;
+}
+
+.toggle-button-off {
+    background-color: red;
+}
+.toggle-button {
+    position: absolute;
+    z-index: 1000;
+    height: 45px;
+    width: 45px;
+    border-radius: 50%;
+    text-align: center;
+    overflow-wrap: break-word;
```

Comment:
I don't think you need to set `text-align` and `overflow-wrap`. (I like to keep these samples as terse as possible.)